docs/backend-requests/making/custom-session-token.mdx

```diff
@@ -7,7 +7,7 @@ description: Learn how to customize the session token that is generated for you
 
 Session tokens are JWTs generated by Clerk on behalf of your instance, and convey an authenticated user session to your backend.
 
-Session tokens typically contain a standard set of claims that are required for Clerk to function. You can learn more about these *default claims* in the [session tokens](/docs/backend-requests/resources/session-tokens) documentation.
+Session tokens typically contain a standard set of claims that are required for Clerk to function. You can learn more about these "default claims" in the [session tokens](/docs/backend-requests/resources/session-tokens) documentation.
```

Comment:
```suggestion
By default, session tokens contain claims that are required for Clerk to function. You can learn more about these "default claims" in the [session tokens](/docs/backend-requests/resources/session-tokens) documentation.
```
Suggesting this change because saying "typically" with no explanation feels a bit confusing and makes the reader wonder, "When do they not contain these standard claims?"